controllers/service.go

```diff
@@ -182,6 +146,23 @@ func (r *SingleClusterReconciler) createSTSLoadBalancerSvc() error {
 		return err
 	}
 
+	if len(service.Spec.Ports) == 1 && service.Spec.Ports[0].Port == servicePort.Port &&
```

Comment:
There are other fields also in LB struct like `LoadBalancerSourceRanges`, `ExternalTrafficPolicy` and `Annotations`. Shouldn't we handle those as well as part of update check?

Reply:
validation web hook does not allow any modification in loadbalancer spec.

Reply:
As per discussion, not allowing LB service update through CR for now